idaplugin/rematch/actions/match.py

```diff
@@ -16,8 +16,7 @@ class MatchAction(base.BoundFileAction):
   name = "&Match"
   dialog = MatchDialog
 
-  def __init__(self, *args, **kwargs):
-    super(MatchAction, self).__init__(*args, **kwargs)
+  def reset(self):
     self._running = False
```

Comment:
When actions are executed the 2nd time, we need to reset their state. It might be easier to do using a `reset` method.